src/Routes/SandboxPage/SandboxPage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import DescriptionCard from '../../Components/PresentationalCard/DescriptionCard';
+import PurchaseCard from '../../Components/PurchaseCard/PurchaseCard';
+import TopBanner from '../../Components/TopBanner/TopBanner';
+import Main from '@redhat-cloud-services/frontend-components/Main';
+import './SandboxPage.scss';
+
+const SandboxPage = () => {
+  return (
+    <>
```

Comment:
No need for extra return statement, can we `Fragment` instead of emtpy element? It's more readable.
```suggestion
const SandboxPage = () => (
  <Fragment>
```